src/pages/contributors/ilan-rabinovitch.adoc

```diff
@@ -0,0 +1,172 @@
+Similarly, in the case of SCaLE there's lots of code that we write.
+For example, our registration system is open source and the software used to power our network is 100% open source.
+We got to work with the link:https://sfconservancy.org/[Software Freedom Conservancy] this year to make sure that even the hardware that powers the network is open as well.
+
+We're using their OpenWrt Ones this year, which is their open hardware project that runs open-source software, that we then use to enable open-source developers to get online at SCaLE.
+So we do a lot of code in that sense, but there's also the person that has to run a network cable from the first floor to the second floor so that the workshop can be online, or the team that does our AV setup, or the greeter that stands at the door to welcome attendees when they show up for the first time.
+All of these are important, just like there's lots of roles in an open-source project that are important, even if they don't involve code.
+All of these things fit together to make a community successful, and I think SCaLE sort of reinforced that.
+
+The other thing that's been fun to see in SCaLE over the last twenty-odd years has been kids that were showing up to SCaLE 15 years ago, and now running into them in their careers outside in the tech world.
+One of the things that we do at SCaLE, and have done for many years, is create opportunities for youth to show off how they're using open source and the interesting things that they're doing in tech, both in terms of demoing it and in terms of speaking about it in talks.
+So we have pre-teens and teenagers showing up at SCaLE and giving 45 minute talks about how they did something with a GPU, Linux, or whatever it might be.
+I've now run into kids that were 12 years old 12 years ago, and now they're 24 and working as engineers or product managers, or in some cases even founders of companies.
+
+For example I was at one of the last KubeCons for work purposes, and as I'm walking down the expo floor, this young gentleman runs up to me and says "Ilan do you remember me?"
+I couldn't remember and told him that I felt bad not recalling, but he shared that he had given a talk at SCaLE back when he was 12.
+I said "you probably look much different in that case" and found out that he's in a leadership role in a very successful tech company.
+I think what this has taught me about organizations and open-source communities, is that if you create these on-ramps, onboarding opportunities, and these educational opportunities, it's like planting seeds in a garden where it eventually turns into a forest that becomes self-sustaining.
+If you don't create those things, the community doesn't sort of re-seed itself and refresh itself, so that's something that we need to apply to our open source projects as well.
```

Comment:
```suggestion
If you don't create those things, the community doesn't sort of re-seed itself and refresh itself, so that's something that we need to apply to our open-source projects as well.
```